lib/pages/home/home_courses_page.dart

```diff
@@ -37,21 +38,50 @@ class _HomeCoursesPageState extends State<HomeCoursesPage> {
               margin: const EdgeInsets.only(top: 16),
               child: Column(
                 crossAxisAlignment: CrossAxisAlignment.start,
-                children: const [
-                  CourseCard(
-                    color: Color.fromRGBO(251, 146, 60, 1),
-                    title: 'IHC y Tecnología Móviles',
-                    location: 'Online',
-                    time: 'Quedan 30 minutos',
+                children: [
+                  InfoCard(
+                    backgroundColor: const Color.fromRGBO(251, 146, 60, 1),
+                    foregroundColor: Colors.white,
+                    children: [
+                      InfoCardItem(
+                        child: Text(
+                          'IHC y Tecnología Móviles',
+                          style:
+                              textTheme.headlineSmall?.withColor(Colors.white),
```

Comment:
`InfoCard`s already apply text styles and colors correctly, so this is not needed.

Reply:
aight